docs/tutorials/operations.md

```diff
@@ -0,0 +1,166 @@
+
+A minimal set of arguments that should be chosen to support most
+Kubernetes use cases:
+
+```
+          - "skipper"
+          - "-kubernetes"
+          - "-kubernetes-in-cluster"
+          - "-kubernetes-path-mode=path-prefix"
+          - "-address=:9999"
+          - "-wait-first-route-load"
+          - "-proxy-preserve-host"
+          - "-enable-ratelimits"
+          - "-experimental-upgrade"
+          - "-lb-healthcheck-interval=3s"
+          - "-metrics-flavour=prometheus"
+          - "-metrics-exp-decay-sample"
+          - "-serve-host-metrics"
+          - "-disable-metrics-compat"
+          - "-enable-connection-metrics"
+          - "-histogram-metric-buckets=.0001,.00025,.0005,.00075,.001,.0025,.005,.0075,.01,.025,.05,.075,.1,.2,.3,.4,.5,.75,1,2,3,4,5,7,10,15,20,30,60,120,300,600"
+          - "-max-audit-body=0"
+          - "-idle-timeout-server=62s"
+```
+
+Skipper started with these options will support instance based
+ratelimits, a wide range of Prometheus metrics, websockets and a
+better HTTP path routing than the default Kubernetes Ingress spec
+supports.
+
+The Kubernetes Ingress spec defines a
+[path](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#httpingresspath-v1beta1-networking-k8s-io)
+as regular expression, which is not what most people would expect, nor
+want. Skipper defaults in Kubernetes to use the [PathRegexp predicate](../../reference/predicates.md#pathregexp)
+for routing, because of the spec. We believe the better default is the
+path prefix mode, that uses [PathSubtree predicate](../../reference/predicates.md#pathsubtree),
+instead. Path prefix search is much more scalable and can not lead to
+unexpected results by not so experienced regular expressions users.
+
+To find more information about Metrics, including formats and example
+Prometheus queries you find in the [metrics
+section](../../operation/operation.md#monitoring).
+The settings shown above support system and application metrics to
+carefully monitor Skipper and your backend applications. Backend
+application metrics get error rates and latency buckets based on host
+headers. The chosen options are a good setup to safely run all
+workloads from small to high traffic.
+
+The option `-max-audit-body=0`, won't log the HTTP body, if you would
+do audit logging, to have a safe default.
+
+The last option `-idle-timeout-server=62s` was chosen, because of a
+[known issue](https://github.com/zalando/skipper/issues/964), if you
+run in a multi layer loadbalancer, with ALBs in front of Skipper.
+[ALBs idle connection timeout is 60s](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#connection-idle-timeout)
+and AWS support told us to run the backends with a bigger timeout,
+than the ALB in front.
+
+### Opt-In more features
+
+Depending on the HTTP loadbalancer in front of your Skippers, you might
+want to set `-reverse-source-predicate`. This setting reverse the
```

Comment:
"This setting reverse the..." -> "This setting reverses the..."

Reply:
fixed